src/java.base/share/classes/java/lang/constant/DynamicCallSiteDesc.java

```diff
@@ -41,12 +41,9 @@
  * A <a href="package-summary.html#nominal">nominal descriptor</a> for an
  * {@code invokedynamic} call site.
  *
- * <p>Concrete subtypes of {@linkplain DynamicCallSiteDesc} should be immutable
- * and their behavior should not rely on object identity.
```

Comment:
Please reword this to something like:
```
{@code DynamicCallSiteDesc} is immutable and its behavior does not rely on object identity.
```
This is given in `ConstantDesc`, but `DynamicCallSiteDesc` does not extend `ConstantDesc` so the removal is dubious.

Reply:
Thanks, I've updated the PR and CSR to retain the note about immutability and object identity, removing the reference to subtypes:

```
This class is immutable and its behavior does not rely on object identity
```

I opted to replace the self-reference `{@code DynamicCallSiteDesc}` with just "This class" here. Let me know if you prefer spelling out the class name.

Reply:
For the record:

It was not completely clear to me whether restricting these two package-private constructors to private is considered a specification change in the CSR sense. I opted to point them out in the CSR, just to make the change abundantly clear.